src/isomorphic/classic/element/ReactCurrentOwner.js

```diff
@@ -11,20 +11,6 @@
 
 'use strict';
 
-/**
- * Keeps track of the current owner.
- *
- * The current owner is the component who should own any components that are
- * currently being constructed.
- */
-var ReactCurrentOwner = {
-
-  /**
-   * @internal
-   * @type {ReactComponent}
-   */
-  current: null,
-
-};
+var ReactCurrentOwner = require('react-current-owner');
```

Comment:
This is just a matter of having to do special configs for these things in our website code to work. Nothing else to do about it.

In this code base you should be able to make it work - we have to hard code that `react-current-owner` is normal and should not get rewritten. And you have to do that in the gulpfile (again, sorry for the crazy build system). https://github.com/facebook/react/blob/master/gulpfile.js#L42 - we should make that `Object.assign({}, require('fbjs/module-map'), {'react-current-owner': 'react-current-owner'})`. Basically the build assumes that if something isn't on the whitelist, then it must be a providesModule module. We dump all our modules into lib/ and rewrite the requires so they are relative. So this is currently getting rewritten to `require('./react-current-owner')` which is going to fail when browserifying.